### DISCUSSION TITLE
401 from magic link request

### DISCUSSION THREAD

**Discussion by jdcauley** [supabase/supabase#3950] [Questions]
I'm trying magic link auth on a prod url, I've added all the urls we might load magic link from in my redirect allow list. After I submit an email through the supabase client functions I get no api key included, and the network request supports that.

If the supabase client is loading how could it be submitting a magic link without a token?

**Comment by haydn** [comment]
Could you be hitting [this "Error inviting user" bug](https://github.com/supabase/supabase/issues/3948) too?

**Reply by jdcauley** [reply]
I don't think so but maybe.


This did just kind of go away as a problem the next day, maybe it was a caching problem?
